userdatamodel/user.py

```diff
@@ -50,20 +50,17 @@ def __setitem__(self, key, value, _sa_initiator=None):
 
 
 class User(Base):
-
     __tablename__ = "User"
-
     id = Column(Integer, primary_key=True)
     username = Column(String(255), unique=True)
-
+    mfa_status = Column(Boolean, default=False)
```

Comment:
could you add this to the PR description

Reply:
I've removed this from the changes.